templates/essentials.yaml

```diff
@@ -387,11 +388,16 @@ Resources:
             Principal:
               Service:
                 - "ec2.amazonaws.com"
+                #Needed for SSM management
+                - "ssm.amazonaws.com"
             Action:
               - "sts:AssumeRole"
       Path: "/"
       ManagedPolicyArns:
         - !Ref TagRootVolumePolicy
+          #Add policy to role allow SSM to attach to instances
+        - 'arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore'
+# This profile would more accurately be named "EC2ManagementProfile"
```

Comment:
I believe this change will re-provision existing instances.

How about we create a new set of `EC2ManagementProfile`, `EC2ManagementRole` and `EC2ManagementPolicy`.  Then selectively apply this to specific templates?

Reply:
We won't need a `EC2ManagementPolicy` for this step as I'm using an AWS managed policy, but I agree, this should be separated into a new role to prevent a potentially destructive change